integration-tests/features/importing_coverage.feature

```diff
@@ -93,33 +93,6 @@ Feature: Importing coverage data
               | overall_line_coverage   | 13.7  |
               | overall_branch_coverage | 50    |
 
-          
-  @SqApi62
-  Scenario: Importing coverage reports zeroing coverage for untouched files
-      GIVEN the project "coverage_project"
-
-      WHEN I run sonar-scanner with following options:
-          """
-          -Dsonar.cxx.coverage.reportPath=ut-coverage.xml
-          -Dsonar.cxx.coverage.itReportPath=it-coverage.xml
-          -Dsonar.cxx.coverage.overallReportPath=overall-coverage.xml
-          -Dsonar.cxx.coverage.forceZeroCoverage=True
```

Comment:
Why do you exclude this test? For SQ 6.2 it should work without `-Dsonar.cxx.coverage.forceZeroCoverage=True`?

Reply:
I think this is already covered with "Importing coverage reports" and the check for 0% coverage is still executed (scenario: _"Zero coverage measures without coverage reports"_).  Maybe the property "-Dsonar.cxx.coverage.forceZeroCoverage=False" should be removed from this scenario as well.